go/bridge/bridge_test.go

```diff
@@ -41,6 +43,11 @@ func TestServe(t *testing.T) {
 		t.Fail()
 	}
 
+	if res.Headers["X-RequestURI"][0] != "/path" {
+		fmt.Printf("expected response header \"X-RequestURI\" == \"%s\"\n", res.Headers["X-RequestURI"][0])
+		t.Fail()
```

Comment:
This fails and I don't understand why:

```
Running tool: /usr/local/go/bin/go test -timeout 30s -run ^TestServe$ bridge/go/bridge

=== RUN   TestServe
--- FAIL: TestServe (0.00s)
panic: runtime error: index out of range [0] with length 0 [recovered]
        panic: runtime error: index out of range [0] with length 0

goroutine 18 [running]:
testing.tRunner.func1.2({0x1409000, 0xc0001d0000})
        /usr/local/go/src/testing/testing.go:1396 +0x24e
testing.tRunner.func1()
        /usr/local/go/src/testing/testing.go:1399 +0x39f
panic({0x1409000, 0xc0001d0000})
        /usr/local/go/src/runtime/panic.go:884 +0x212
bridge/go/bridge.TestServe(0xc000182680)
        /Users/smassa/source/vercel/go-bridge/go/bridge/bridge_test.go:46 +0x779
testing.tRunner(0xc000182680, 0x1463968)
        /usr/local/go/src/testing/testing.go:1446 +0x10b
created by testing.(*T).Run
        /usr/local/go/src/testing/testing.go:1493 +0x35f
FAIL    bridge/go/bridge        0.403s
```

Reply:
If I print out the value of `r.RequestURI` inside `ServeHTTP`, I see the correct value. It just doesn't appear in the response headers for the test.

Reply:
Turns out it was a casing issue. The header is rewritten to `X-Requesturi`.